run-spill-build/overlaySinglesIntoSpillsSorted.C

```diff
@@ -52,37 +60,81 @@ void overlaySinglesIntoSpillsSorted(std::string inFileName1,
                                     double spillPOT = 5E13,
                                     double spillPeriod_s = 1.2) {
 
-  // get input nu-LAr files
-  TChain* edep_evts_1 = new TChain("EDepSimEvents");
-  edep_evts_1->Add(inFileName1.c_str());
+  // Check that we are considering, nu-LAr only, nu-rock only or both.
+  if (inFile1POT==0. && inFile2POT==0.) {
+    throw std::invalid_argument("nu-LAr POT and nu-rock POT cannot both be zero!")
+  }
+  // "N Interaction" mode only supported in nu-LAr mode. Choice of 100 here 
```

Comment:
Very non-critical comment here: but I would actually consider moving this to something like 1K to 10K in case we wanted to stress test the pileup handling. Either way we should be well below an actual POT/spill number which should be at least O(trillion)

Reply:
Agreed - addressed in ddb62a23c6f04522081c2ce932caf10399dd4dd7. Have made the number a variable so we can switch it in just one place if needs be.